README.md

```diff
@@ -1 +1,104 @@
-# ansible-letsencrypt
\ No newline at end of file
+# Ansible Letsecrypt
+
+[![Stack](https://raw.githubusercontent.com/paralect/stack/master/stack-component-template/stack.png)](https://github.com/paralect/stack)
+
+[![All Contributors](https://img.shields.io/badge/all_contributors-2-orange.svg?style=flat-square)](#contributors)
+[![Ansible Galaxy](https://img.shields.io/badge/ansible--galaxy-letsenctypt-blue.svg?style=flat-square)](https://galaxy.ansible.com/paralect/drone)
+[![license](https://img.shields.io/github/license/mashape/apistatus.svg?style=flat-square)](https://github.com/paralect/ansible-mongo/blob/master/LICENSE)
+[![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg?style=flat-square)](http://makeapullrequest.com)
+
+
+[![Watch on GitHub](https://img.shields.io/github/watchers/paralect/ansible-letsectrypt.svg?style=social&label=Watch)](https://github.com/paralect/ansible-letsectrypt/watchers)
+[![Star on GitHub](https://img.shields.io/github/stars/paralect/ansible-letsectrypt.svg?style=social&label=Stars)](https://github.com/paralect/ansible-letsectrypt/stargazers)
+[![Follow](https://img.shields.io/twitter/follow/paralect.svg?style=social&label=Follow)](https://twitter.com/paralect)
+[![Tweet](https://img.shields.io/twitter/url/https/github.com/paralect/ansible-letsectrypt.svg?style=social)](https://twitter.com/intent/tweet?text=I%27m%20using%20Stack%20components%20to%20build%20my%20next%20product%20🚀.%20Check%20it%20out:%20https://github.com/paralect/ansible-letsectrypt)
+
+The ansible role for generating [letsecrypt](https://letsencrypt.org/) certificates.
```

Comment:
I think we need to add more explanation here. We need to mention that (we need to add feature list with emotions, just like on other repos):
 - role can generate single certificates for specific domains/subdomains
 - role can generate wildcard certificates
 - I am not sure what this role do with DNS provider (Cloudflare for example) and why it need it, but we need to mention this as well :)
- we need to mention that certificates will automatically renew every month (or so)
- we need to mention where generated certificates stored and how their naming works.